third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/TensorMemoryToLLVM.cpp

```diff
@@ -39,19 +41,222 @@ constexpr TMemAccessAtom TMemAccess32x32b{.opBitWidth = 32,
                                           .opShape = "32x32b",
                                           .usesSecondHalfOffset = false};
 
+constexpr TMemAccessAtom TMemAccess16x256b{.opBitWidth = 256,
+                                           .colsPerThread = 2,
+                                           .rowsPerThread = 2,
+                                           .rowStored = 16,
+                                           .opShape = "16x256b",
+                                           .usesSecondHalfOffset = false};
+
 constexpr TMemAccessAtom TMemAccess16x32bx2{.opBitWidth = 32,
                                             .colsPerThread = 1,
                                             .rowsPerThread = 1,
                                             .rowStored = 32,
                                             .opShape = "16x32bx2",
                                             .usesSecondHalfOffset = true};
 
-constexpr TMemAccessAtom TMemAccess16x256b{.opBitWidth = 256,
-                                           .colsPerThread = 2,
-                                           .rowsPerThread = 2,
-                                           .rowStored = 16,
-                                           .opShape = "16x256b",
-                                           .usesSecondHalfOffset = false};
+static std::optional<LinearLayout> getReps(const LinearLayout &cvt,
+                                           const LinearLayout &tile) {
```

Comment:
nb. this is a bit of a prototype.  I might factor this out in a follow-up PR into LayoutUtils and use it in a few other places.